pkg/osutil/user.go

```diff
@@ -141,7 +141,7 @@ func LimaUser(limaVersion string, warn bool) *user.User {
 				warnings = append(warnings, warning)
 				limaUser.Gid = formatUidGid(gid)
 			}
-			home, err := call([]string{"cygpath", limaUser.HomeDir})
+			home, err := call([]string{"realpath", "~"})
```

Comment:
Does this depend on MSYS2?

Eventually this should use pure Windows API that doesn't depend on MSYS2, Cygwin, or whatever else similar.

Reply:
The deps should be documented in https://lima-vm.io/docs/config/vmtype/#known-issues-1

Reply:
I will check this out.

Reply:
> Does this depend on MSYS2?
realpath is part of the same core package as cygpath. But I will rework it back to cygpath + .linux suffix. This is needed to prevent nuking home dir, when used with QEMU